cyber/message/message_traits_test.cc

```diff
@@ -34,10 +34,10 @@ class Message {
  public:
   std::string content;
 
-  int ByteSize() const { return static_cast<int>(content.size()); }
+  int ByteSizeLong() const { return static_cast<int>(content.size()); }
```

Comment:
It looks confusing to me that `ByteSizeLong` returns `int` instead of `long`. Any reason to change the name?

Reply:
> It looks confusing to me that `ByteSizeLong` returns `int` instead of `long`. Any reason to change the name?

Because the protobuf v3.12.3 "ByteSize()" function have a compilation attribute "deprecated", it can warn us the function is out of date and recommand to use "ByteSizeLong()", which is different from the protobuf v3.3.0. So I use the "ByteSizeLong()" to match the template function "cyber/message/message_traits.h: ByteSize()".